osmaxx/excerptexport/templates/excerptexport/partials/export_display.html

```diff
@@ -31,5 +30,11 @@
                 </div>
             </div>
         </div>
+    {% elif export.status == "finished" %}{# FIXME: finished is hardcoded, but as long as we don't have the STATUS on the model, we can't access it here! #}
```

Comment:
:bulb: `import` the stati in [the model class](https://github.com/geometalab/osmaxx-frontend/pull/513/files#diff-e52c21d719a322835dc4db89ea595076R30) to make them accessible on the model instances.